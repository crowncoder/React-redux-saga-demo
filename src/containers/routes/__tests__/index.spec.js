import React from 'react';
import { render } from 'enzyme';
import LoginContainer from '../';
import configureStore from '../../../stores';
const store = configureStore();
it('Render Login Container', () => {
	const wrapper = render(<LoginContainer store={store} />);
	expect(wrapper).toMatchSnapshot();
});
