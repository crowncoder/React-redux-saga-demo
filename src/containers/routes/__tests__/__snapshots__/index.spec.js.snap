// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Login Container 1`] = `
<div
  class="loginWrapper"
>
  <div
    class="loginBody"
  >
    <img
      class="loginImg"
      src="[object Object]"
    />
    <form
      class="ant-form ant-form-horizontal loginForm"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control-wrapper"
        >
          <div
            class="ant-form-item-control"
          >
            <span
              class="ant-form-item-children"
            >
              <span
                class="ant-input-affix-wrapper"
              >
                <span
                  class="ant-input-prefix"
                >
                  <i
                    aria-label="icon: user"
                    class="anticon anticon-user"
                    style="font-size:13px"
                  >
                    <svg
                      aria-hidden="true"
                      class=""
                      data-icon="user"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M858.5 763.6a374 374 0 0 0-80.6-119.5 375.63 375.63 0 0 0-119.5-80.6c-.4-.2-.8-.3-1.2-.5C719.5 518 760 444.7 760 362c0-137-111-248-248-248S264 225 264 362c0 82.7 40.5 156 102.8 201.1-.4.2-.8.3-1.2.5-44.8 18.9-85 46-119.5 80.6a375.63 375.63 0 0 0-80.6 119.5A371.7 371.7 0 0 0 136 901.8a8 8 0 0 0 8 8.2h60c4.4 0 7.9-3.5 8-7.8 2-77.2 33-149.5 87.8-204.3 56.7-56.7 132-87.9 212.2-87.9s155.5 31.2 212.2 87.9C779 752.7 810 825 812 902.2c.1 4.4 3.6 7.8 8 7.8h60a8 8 0 0 0 8-8.2c-1-47.8-10.9-94.3-29.5-138.2zM512 534c-45.9 0-89.1-17.9-121.6-50.4S340 407.9 340 362c0-45.9 17.9-89.1 50.4-121.6S466.1 190 512 190s89.1 17.9 121.6 50.4S684 316.1 684 362c0 45.9-17.9 89.1-50.4 121.6S557.9 534 512 534z"
                      />
                    </svg>
                  </i>
                </span>
                <input
                  class="ant-input"
                  data-__field="[object Object]"
                  data-__meta="[object Object]"
                  id="userName"
                  placeholder="Username"
                  type="text"
                  value=""
                />
              </span>
            </span>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control-wrapper"
        >
          <div
            class="ant-form-item-control"
          >
            <span
              class="ant-form-item-children"
            >
              <span
                class="ant-input-affix-wrapper"
              >
                <span
                  class="ant-input-prefix"
                >
                  <i
                    aria-label="icon: lock"
                    class="anticon anticon-lock"
                    style="font-size:13px"
                  >
                    <svg
                      aria-hidden="true"
                      class=""
                      data-icon="lock"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M832 464h-68V240c0-70.7-57.3-128-128-128H388c-70.7 0-128 57.3-128 128v224h-68c-17.7 0-32 14.3-32 32v384c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V496c0-17.7-14.3-32-32-32zM332 240c0-30.9 25.1-56 56-56h248c30.9 0 56 25.1 56 56v224H332V240zm460 600H232V536h560v304zM484 701v53c0 4.4 3.6 8 8 8h40c4.4 0 8-3.6 8-8v-53a48.01 48.01 0 1 0-56 0z"
                      />
                    </svg>
                  </i>
                </span>
                <input
                  class="ant-input"
                  data-__field="[object Object]"
                  data-__meta="[object Object]"
                  id="password"
                  placeholder="Password"
                  type="password"
                  value=""
                />
              </span>
            </span>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control-wrapper"
        >
          <div
            class="ant-form-item-control has-success"
          >
            <span
              class="ant-form-item-children"
            >
              <label
                class="rememberBox ant-checkbox-wrapper ant-checkbox-wrapper-checked"
              >
                <span
                  class="ant-checkbox ant-checkbox-checked"
                >
                  <input
                    checked=""
                    class="ant-checkbox-input"
                    data-__field="[object Object]"
                    data-__meta="[object Object]"
                    id="remember"
                    type="checkbox"
                  />
                  <span
                    class="ant-checkbox-inner"
                  />
                </span>
                <span>
                  Remember me
                </span>
              </label>
              <a
                class="loginFormForgot"
                href=""
              >
                Forgot password
              </a>
              <button
                class="ant-btn loginFormButton ant-btn-primary"
                type="submit"
              >
                <span>
                  Log in
                </span>
              </button>
            </span>
          </div>
        </div>
      </div>
    </form>
  </div>
</div>
`;
